src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -100,28 +104,41 @@ public void ActiveSolutionBoundTracker_Bound()
 
             // Assert
             testSubject.IsActiveSolutionBound.Should().BeTrue("Bound solution should report true activation");
-            this.errorListController.RefreshCalledCount.Should().Be(1);
+            // TODO: this.errorListController.RefreshCalledCount.Should().Be(1);
```

Comment:
We aren't currently calling the the _Refresh_ method from the tracker. There is a _TODO_ in the tracker, and I've put _TODOs_ in the relevant tests.